src/pmdas/amdgpu/pmns

```diff
@@ -0,0 +1,33 @@
+/*
+ * Metrics for amd GPU PMDA
+ *
+ * Copyright (c) 2024 Red Hat.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation; either version 2 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+ * or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * for more details.
+ */
+
+amdgpu {
+    numcards			AMDGPU:0:0
+    cardname			AMDGPU:0:1
+    memused			AMDGPU:0:2
+    memtotal			AMDGPU:0:3
+    memfree			AMDGPU:0:4
+    samples			AMDGPU:0:5
+    memusedaccum		AMDGPU:0:6
+    mem_clock			AMDGPU:0:7
+    mem_clock_max		AMDGPU:0:8
+    gpu_clock			AMDGPU:0:9
+    gpu_clock_max		AMDGPU:0:10
+    temperature			AMDGPU:0:11
```

Comment:
With several of the above metrics, consider a "memory" subtree in the PMNS, e.g. amdgpu.memory.clock and so on (total, free, etc).  Possibly same for amdgpu.gpu.clock and clock_max.